### ISSUE TITLE
Ture skrives fejlagtigt ud

### ISSUE THREAD

**Issue by Husmaar** [Husmaar/ARK-site#8]
Af og til er ens tur skrevet ud når man kommer. Måske er det et dobbeltklik problem?

**Comment by StephanLorenzen** [comment]
Har altid troet det skyldste at en anden roer havde skrevet en ud ved en fejl?

**Comment by Husmaar** [comment]
Ja, men jeg synes det sker for ofte til at jeg tror det er grunden. Jeg har en mistanke om at det skyldes at der er nogen der dobbeltklikker i stedet for enkelt klikker. Jeg har godt nok lavet noget der skulle håndtere det, men måske virker det ikke rigtigt. Hvis du oplever det så skriv turdata ind her. Så er der noget at debugge på.

**Comment by StephanLorenzen** [comment]
Skete i dag, min tur startet kl 12.00 i dag (2019-06-23) var skrevet ind, da jeg kom tilbage ca. 13.40.
